### PR TITLE
feat: add mai icon and name

### PR DESCRIPTION
**Pull request by Argeare5**
prepared FRAX and MAI for Avalanche

### CODE REVIEW

**Review by sendra — APPROVED** [review]

File: src/ui-config/reservePatches.ts [hunk 1]
```diff
@@ -35,6 +35,8 @@ export const SYMBOL_MAP: { [key: string]: string } = {
   'AAVE.e': 'AAVE',
   'USDT.e': 'USDT',
   'USDC.e': 'USDC',
+  'FRAX.e': 'FRAX',
```
Comment (sendra) [inline src/ui-config/reservePatches.ts:38]: symbol of Frax on avalanche is FRAX https://snowtrace.io/address/0xD24C2Ad096400B6FBcd2ad8B24E7acBc21A1da64#readContract

symbol of Mai is MAI on avalanche: https://snowtrace.io/address/0x5c49b268c9841aff1cc3b0a418ff5c3442ee3f3b#readContract

Comment (Argeare5) [reply src/ui-config/reservePatches.ts]: ok, deleted it
reinsured just in case =)

File: src/ui-config/reservePatches.ts [hunk 2]
```diff
@@ -59,6 +61,7 @@ export const SYMBOL_NAME_MAP: { [key: string]: string } = {
   GUSD: 'Gemini Dollar',
   KNC: 'Kyber Legacy',
   LINK: 'ChainLink',
+  MAI: 'MIMATIC',
```
Comment (sendra) [inline src/ui-config/reservePatches.ts:62]: On polygon MAI symbol is miMATIC: https://polygonscan.com/address/0xa3Fa99A148fA48D14Ed51d610c367C61876997F1#readContract

Comment (Argeare5) [reply src/ui-config/reservePatches.ts:64]: ok, added symbol fix to `SYMBOL_MAP`

**Review by sendra — APPROVED** [review]

**Review by MareskoY — APPROVED** [review]

**Review by foodaka — APPROVED** [review]

**Pull request merged**